kpi/models/asset.py

```diff
@@ -11,10 +11,6 @@
 from django.db import models, transaction
 from django.db.models import F, Prefetch, Q
 from django.utils.translation import gettext_lazy as t
-from taggit.managers import TaggableManager, _TaggableManager
```

Comment:
Accidental change?

Reply:
This may be a rebase artifact. There should be no changes to this file in the PR i'm assuming.